src/views/components/building-address/building-address-input-fields.njk

```diff
@@ -0,0 +1,75 @@
+{% from "hmpo-text/macro.njk" import hmpoText %}
+{%- from "govuk/components/hint/macro.njk" import govukHint %}
+{%- from "govuk/components/label/macro.njk" import govukLabel %}
+{% from "./macro.njk" import customInput %}
+
+{%- set args = {
+  id: "buildingAddress",
```

Comment:
We are going to be implementing this in the UK address form soon, so maybe we want to rename this one to indicate it's international?

Reply:
Can that be done when we do that task, I have already broken this up based on @mikebeeby  comments so we have a `address-input-fields` and `non-UK-address-input-fields`